ReportingServicesTools/Functions/CatalogItems/Rest/Get-RsRestItemDataSources.ps1

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Get-RsRestItemDataSources
+{
+    <#
+        .SYNOPSIS
+            This script fetches data sources related to a catalog item from the Report Server
+
+        .DESCRIPTION
+            This script fetches data sources related to a catalog item from the Report Server
+
+        .PARAMETER RsItem
+            Specify the location of the catalog item whose data sources should be fetched.
+
+        .PARAMETER ReportPortalUri
+            Specify the Report Portal URL to your SQL Server Reporting Services Instance.
+
+        .PARAMETER RestApiVersion
+            Specify the version of REST Endpoint to use. Valid values are: "v2.0".
+
+        .PARAMETER Credential
+            Specify the credentials to use when connecting to the Report Server.
+
+        .PARAMETER WebSession
+            Specify the session to be used when making calls to REST Endpoint.
+
+        .EXAMPLE
+            Get-RsRestItemDataSources -RsItem "/MyReport"
+
+            Description
+            -----------
+            Fetches data sources associated to "MyReport" catalog item found in "/" folder.
+    #>
+
+    [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'Medium')]
+    param(
+        [Parameter(Mandatory = $True)]
+        [Alias('ItemPath','Path')]
```

Comment:
Alias would be optional for new functions, at least you want to keep it in parity with the non rest version